.golangci.yml

```diff
@@ -5,7 +5,7 @@ linters:
   disable-all: true
   enable:
     - gofmt
-    - golint
```

Comment:
Let's not remove golint

Reply:
All errors are not cleared with golint at, hence commented for now, let me know if you want me to enable ?

Reply:
Let's enable it - we'are still not blocking on it for nw.